src/rez/tests/test_build.py

```diff
@@ -155,6 +156,11 @@ def test_builds_anti(self):
     @program_dependent("cmake")
     def test_build_cmake(self):
         """Test a cmake-based package."""
+        if platform_.name == "windows":
```

Comment:
I'm wondering if we could potentially do:

```python
    @unittest.skipIf(platform_.name == "windows", "This ...")
    @program_dependent("cmake")
    def test_build_cmake(self):
        """Test a cmake-based package."""
```
Would that work?

Reply:
I'm merging this anyway, given that it's a temporary measure. If it works then it works - better to get traction on the source of the problem (and yes Blazej I will get up to speed on that ticket soon).

Reply:
Looks good @JeanChristopheMorinPerso.
I just took it from some other location that does it with a condition, so we might want to unify all of them.
But it's also true that we should try to get rid of these guys completely :)